std/http/io.ts

```diff
@@ -171,6 +172,9 @@ export async function writeChunkedBody(
     await writer.write(start);
     await writer.write(chunk);
     await writer.write(end);
+    if (options && options.realTime) {
+      await writer.flush();
+    }
```

Comment:
Wouldn't it be possible to make flushing functionality available to callers of `writeChunkedBody`? One example would be sending special values through `r` as a command to flush. To make it more elegant, I would say `response.body` should be a `AsyncIterable`.